civihr_default_theme/scss/base/_custom.scss

```diff
@@ -3,16 +3,17 @@ legend {
   border-bottom: 1px solid $legend-border-color;
   color: $legend-color;
   display: block;
-  font-size: ($font-size-base * 1.5);
+  font-size: $font-size-base;
   line-height: inherit;
   margin-bottom: $line-height-computed;
   padding: 0;
   width: 100%;
 }
 
 label {
+  color: $gray-light;
```

Comment:
what was the color before?

Reply:
According to new mockup, the labels should be `#9494a5`($gray-light). Previously the label's were using body color(`#757b8e`) when inside the modal's. So defining this globally for all labels.